### ISSUE TITLE
Speed multiplier settings in ddraw.ini have no effect

### ISSUE THREAD

**Issue by Tinman-6** [BGforgeNet/Fallout2_Restoration_Project#315]
### What happened
Installed fallout 2 RPU on a fresh GOG fallout 2 installation, with all optional files. Character movement speed (out of combat) in game was painfully slow. Tried adjusting the speed setting in the in game settings (including checking the box for the player), no effect (wasn't expecting any since that is combat speed). Tried adjusting the initial speed setting in ddraw.ini up and down (10, 100, 150, 300, 900) all with no effect in game. Tried setting a toggle key for the speed, no effect in game.

ddraw file attached.

### What you expected to happen
Play run speed (and other affected animations) to increase in game.

[ddraw.zip](https://github.com/user-attachments/files/17264618/ddraw.zip)

**Comment by NovaRain** [comment]
If by "slow" you mean the game is super laggy, check if your graphics driver applies any FPS limit or vsync to the game.

**Comment by Tinman-6** [comment]
I appreciate the quick response.

Scrolling by moving the mouse to the sides of the screen was a little
jerky, but not bad. Mainly it is the run animation speed does not seem to
be changing at all, and is painfully slow. Otherwise the game is running
very smooth and fast.

I have a gtx 1080 with a 144hz gsync monitor. Running the game in
borderless windowed mode. I had all my normal driver settings for VRR gsync
in windows applied (was using defaults since i hadn't set a game profile in
the driver settings). I tried turning off gsync, vsync, and any latency
reducing features and no change to the run animation speed in game.

I tried full screen before this, no change to the running animation.

Tried every combination of settings in the configuration files that I
could think of and no change to the running animation speed.
